fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -1738,9 +1738,10 @@ protected UserTransaction retrieveUserTransaction(String jndiName) {
         final String METHODNAME = "convertResourceDTO";
         log.entering(CLASSNAME, METHODNAME);
         T resource = null;
+        InputStream in = null;
         try {
```

Comment:
Why not a try with resources?

Reply:
see https://github.com/IBM/FHIR/pull/2042#issuecomment-794759052